docs/plugin/panel.md

```diff
@@ -118,6 +120,19 @@ values:
 mode: <enum = "auto" | "categorical">
 ```
 
+### Query Settings specification
+
+```yaml
+# queryIndex is an unsigned integer that should match an existing index in the panel's `queries` array
+queryIndex: <number>
```

Comment:
FYI @Nexucis in the end we dropped using a JSON reference here, a basic number to save only the index part was enough on the technical side. Are you fine with this?

Reply:
Yes I am fine with that